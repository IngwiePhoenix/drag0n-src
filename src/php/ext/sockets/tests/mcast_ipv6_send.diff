013+ 
014+ Warning: socket_set_option(): unable to set socket option [49]: Can't assign requested address in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/mcast_ipv6_send.php on line 26
015+ bool(false)
013- bool(true)