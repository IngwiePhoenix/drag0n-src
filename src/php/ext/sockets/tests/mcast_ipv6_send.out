Setting IPV6_MULTICAST_TTL
bool(true)
int(9)

Setting IPV6_MULTICAST_LOOP
bool(true)
int(0)
bool(true)
int(1)

Setting IPV6_MULTICAST_IF
interface 0:

Warning: socket_set_option(): unable to set socket option [49]: Can't assign requested address in /Users/Ingwie/Work/drag0n-php/ext/sockets/tests/mcast_ipv6_send.php on line 26
bool(false)
int(0)
interface 1:
bool(true)
int(1)